lifetimes/utils.py

```diff
@@ -261,24 +261,16 @@ def _scale_time(age):
     return 10. / age.max()
 
 
-def _check_inputs(frequency, recency, T):
-
-    def check_recency_is_less_than_T(recency, T):
```

Comment:
Not sure why these were all nested functions... As far as I can tell, it works just as well without the functions, and the error strings serve as documentation of the error conditions (in case documentation was the original intent).